src/SqlBatis.Schemas/SqlBatis.Schemas.csproj

```diff
@@ -16,8 +16,16 @@
 Matching version with SqlBatis.DataMapper
 </PackageReleaseNotes>
     <IncludeBuildOutput>false</IncludeBuildOutput>
```

Comment:
this package is a content-only nuget package.

Reply:
fixed